schemas/workflow.json

```diff
@@ -0,0 +1,142 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "definitions": {
+        "url": {
+            "oneOf": [
+                {
+                    "type": "string",
+                    "minLength": 1,
+                    "pattern": "^https?://.+"
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        "nullable-string": {
+            "oneOf": [
+                {
+                    "type": "string",
+                    "minLength": 1
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        }
+    },
+    "title": "workflow",
+    "description": "A workflow",
+    "type": "object",
+    "required": [
+        "name",
+        "command"
+    ],
+    "properties": {
+        "name": {
+            "title": "name",
+            "description": "A name of the current workflow\nhttps://github.com/EmilySeville7cfg/warp-workflows/blob/main/FORMAT.md#name",
```

Comment:
Would you mind updating all of these links to point at the warpdotdev repo instead of your fork of this repo?https://github.com/warpdotdev/workflows/blob/main/FORMAT.md